README.md

```diff
@@ -27,6 +27,7 @@ Examples converted to Hummingbird 2.0
 - [http2](https://github.com/hummingbird-project/hummingbird-examples/tree/main/http2) - Basic application with HTTP2 upgrade added.
 - [jobs](https://github.com/hummingbird-project/hummingbird-examples/tree/main/jobs) - Demonstrating offloading of jobs to another server.
 - [multipart-form](https://github.com/hummingbird-project/hummingbird-examples/tree/main/multipart-form) - HTML form using Multipart form data, using MultipartKit
+- [open-telemetry](https://github.com/hummingbird-project/hummingbird-examples/tree/main/open-telemetry) - Integration with Open Telemetry
```

Comment:
```suggestion
- [open-telemetry](https://github.com/hummingbird-project/hummingbird-examples/tree/main/open-telemetry) - Integration with OpenTelemetry
```

Reply:
I'll get it right one day

Reply:
I had auto-merge so its merged. I'll fix separately